packages/kong-plugin/README.md

```diff
@@ -0,0 +1,90 @@
+# ReadMe Metrics + Kong
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/33762/182927634-2aebeb46-c215-4ac3-9e98-61f931e33583.png" />
+</p>
+
+<p align="center">
+  Track usage of your API and troubleshoot issues faster.
+</p>
+
+With [ReadMe's Metrics API](https://readme.com/metrics) your team can get deep insights into your API's usage. If you're a developer, it takes a few small steps to send your API logs to [ReadMe](http://readme.com). Here's an overview of how the integration works:
+
+- Install this plugin into your Kong Gateway by copying the `kong` directory into your Kong environment or use the provided Dockerfile. see [this](https://docs.konghq.com/gateway/latest/plugin-development/get-started/deploy/) for details.
+- The plugin runs during the log phase of the Kong pipeline and sends ReadMe the details of your API's incoming requests and outgoing responses, with the option for you to redact any private headers using the configuration options.
+- ReadMe uses these request and response details to create an API Metrics Dashboard which can be used to analyze specific API calls or monitor aggregate usage data.
+
+### 📦 Deploying locally
+
+```bash
+# Build kong image with the plugin
+docker build -t kong-readme-plugin:v1 .
+# Run kong with the plugin
+curl -Ls https://get.konghq.com/quickstart |  bash -s -- -r "" -i kong-readme-plugin -t v1
+# Enable the plugin
+curl -isX POST http://localhost:8001/plugins -d name=readme-plugin -d 'config.api_key=<Your API Key>'
```

Comment:
Why port `8001`? What if this is exposed on another port?

Reply:
if they ran the above script, its 8001. I am assuming if they don't follow the script they would know what they are doing.